app/services/zendesk_intake_service.rb

```diff
@@ -149,6 +149,26 @@ def send_final_intake_pdf
     output
   end
 
+  def send_intake_pdf_with_spouse
+    comment_body = <<~BODY
+      Updated 13614-c from online intake - added spouse signature and contact
+
+      Client's provided interview preferences: #{@intake.interview_timing_preference}
```

Comment:
i included the interview preferences and additional info here because they are included in the comment for the completed intake - maybe redundant but seemed good to include since it's not on the form